### PR TITLE
Add a test for "keepalive" handling

### PR DESCRIPTION
**Pull request by yutakahirano**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19326)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/286428929)

**Comment by mnot** [comment]
What are you trying to test for here? HTTP's connection behaviour depends on network conditions, and can't be relied upon. I.e., you can't guarantee that two requests to the same origin will go over the same connection, even if they're one right after the other.

**Comment by yutakahirano** [comment]
> What are you trying to test for here? HTTP's connection behaviour depends on network conditions, and can't be relied upon. I.e., you can't guarantee that two requests to the same origin will go over the same connection, even if they're one right after the other.

This is not a test for HTTP keepalive, but for [the keepalive feature in fetch API](https://fetch.spec.whatwg.org/#ref-for-dom-request-keepalive). Does this answer your question?

**Comment by mnot** [comment]
yes, thanks :)

**Comment by yutakahirano** [comment]
Can anyone review this change?

**Comment by yutakahirano** [comment]
@jakearchibald can you take a look?

File: fetch/api/resources/keepalive-iframe.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+<meta charset="utf-8">
+<script src="/common/utils.js"></script>
+<script>
+const xhr = new XMLHttpRequest();
+window.uuid = token();
+// Use a sync XHR to reset the state.
+xhr.open('GET', `../resources/stash-take.py?key=${uuid}`, false);
```
Comment (ricea) [inline fetch/api/resources/keepalive-iframe.html]: I'm not sure this is useful. If I understand correctly, `uuid` will be different every time this test is run. So there will never be an existing key with value `uuid` and this XHR will do nothing.

Comment (yutakahirano) [reply fetch/api/resources/keepalive-iframe.html]: Sounds reasonable, fixed. Thanks.

**Review by ricea — APPROVED** [review]

**Pull request merged**